### ISSUE TITLE
Cannot understand current state of a project

### ISSUE THREAD

**Issue by brianjgeiger** [CenterForOpenScience/osf.io#2611]
Steps
-------
1. Go to a project
2. Determine if it is currently being worked on, if the data collection is complete, etc.

Expected
------------
You could easily determine if a project is complete or in progress or not started.

Actual
--------
If you can figure that out, it is only by digging around in the wiki and other places.

Proposal
------------
We add a status field to the projects that would allow visitors to understand that the things on any given project are completed or not. Things in progress could have a banner that says that the information found here could be changed. These would carry over to registrations. We would want to determine the types of statuses as well as other useful information, such as a link to the registration of the published version of the project etc.

Inspiration from this is drawn from Thingiverse, which allows you to be very clear about the nature of what is being looked at. This could also help with people using it as a workflow tool.

![screen shot 2015-04-30 at 11 19 30 am](https://cloud.githubusercontent.com/assets/6599111/7416071/dd676050-ef2a-11e4-8e57-f5c3b83ab7f1.png)

**Comment by jlcohoon** [comment]
+1 to this idea!

**Comment by meg3c** [comment]
Good idea.  It might be possible for the author to comment on what she/he is working on--what revisions are likely.
